<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabWidgetPageRaw">
       <property name="accessibleName">
        <string>Raw data</string>
       </property>
       <attribute name="title">
        <string>Raw</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Accéléromètre</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>GPS Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Flux optique</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>Données</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Dernière</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="accRawLayout">
            <item>
             <widget class="QLabel" name="accLastDataLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="accLastTimeLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="accRateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Données</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Dernière</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <layout class="QVBoxLayout" name="GPSPosRawLayout">
            <item>
             <widget class="QLabel" name="GPSPosLastDataLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="GPSPosLastTimeLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="GPSPosRateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="4">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Données</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Dernière</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="5">
           <layout class="QVBoxLayout" name="opticFlowRawLayout">
            <item>
             <widget class="QLabel" name="opticFlowLastDataLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="opticFlowLastTimeLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="opticFlowRateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_34">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Gyroscope</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>GPS DOP</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Données</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Dernière</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QVBoxLayout" name="gyroRawLayout">
            <item>
             <widget class="QLabel" name="gyroLastDataLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="gyroLastTimeLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="gyroRateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Données</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Dernière</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Fréquence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="3">
           <layout class="QVBoxLayout" name="GPSDOPRawLayout">
            <item>
             <widget class="QLabel" name="GPSDOPLastDataLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="GPSDOPLastTimeLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="GPSDOPRateLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>3D Graphs</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabWidgetPageGraphs">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCustomPlot" name="accPlot" native="true"/>
          </item>
          <item row="0" column="1">
           <widget class="QCustomPlot" name="GPSPlot" native="true"/>
          </item>
          <item row="0" column="2">
           <widget class="QCustomPlot" name="opticFlowPlot" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QCustomPlot" name="gyroPlot" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="QCustomPlot" name="GPSErrorPlot" native="true"/>
          </item>
          <item row="1" column="2">
           <widget class="QCustomPlot" name="orientationPlot" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPageHistory">
       <attribute name="title">
        <string>History</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Accéléromètre</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Flux optique</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>GPS Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListView" name="accHistoryListView"/>
          </item>
          <item row="3" column="0">
           <widget class="QListView" name="gyrohistoryListView"/>
          </item>
          <item row="3" column="1">
           <widget class="QListView" name="GPSDOPHistoryListView"/>
          </item>
          <item row="3" column="2">
           <widget class="QListView" name="emptyHistoryListView"/>
          </item>
          <item row="1" column="1">
           <widget class="QListView" name="GPSPosHistoryListView"/>
          </item>
          <item row="1" column="2">
           <widget class="QListView" name="opticFlowHistoryListView"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>Gyroscope</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>GPS DOP</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gyrohistoryListView</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>GPSDOPHistoryListView</tabstop>
  <tabstop>emptyHistoryListView</tabstop>
  <tabstop>GPSPosHistoryListView</tabstop>
  <tabstop>opticFlowHistoryListView</tabstop>
  <tabstop>accHistoryListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
